packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -169,12 +173,21 @@ export const TagSubforumPage = ({ classes }: { classes: ClassesType}) => {
 
   const { tag, loading, error } = useTagBySlug(slug, "TagSubforumFragment");
   
+  // hashContents used to contain the tab, now we put that in the query. But it needs to be backwards compatible with that case.
```

Comment:
We switched from using the hash for the tab to putting that in the query so we could put the commentId in the hash, which is consistent with the rest of the site